File: CS/Custom GridView/MyScrollInfo.cs
using System;
using DevExpress.XtraGrid.Scrolling;

namespace MyXtraGrid
{
    public class MyScrollInfo : ScrollInfo
    {
        public MyScrollInfo(DevExpress.XtraGrid.Views.Base.BaseView view)
            : base(view)
        {

        }

        protected override int ScrollSize
        {
            get
            {
                return 100;
            }
        }

    }
}

File: VB/Custom GridView/MyScrollInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraGrid.Scrolling

Namespace MyXtraGrid
	Public Class MyScrollInfo
		Inherits ScrollInfo
		Public Sub New(ByVal view As DevExpress.XtraGrid.Views.Base.BaseView)
			MyBase.New(view)

		End Sub

		Protected Overrides ReadOnly Property ScrollSize() As Integer
			Get
				Return 100
			End Get
		End Property

	End Class
End Namespace
